AzureWebAPI/Models/RequestBody.cs

```diff
@@ -0,0 +1,8 @@
+namespace AzureWebAPI.Models
+{
+    class RequestBody
+    {
+        public string commandId {get; set;}
```

Comment:
Public members should be named using PascalCase, not camelCase.
```suggestion
        public string CommandId {get; set;}
```